src/components/HrefActions.vue

```diff
@@ -1,7 +1,7 @@
 <template>
   <div>
     <b-button-group class="actions" :vertical="vertical" :size="size" v-if="href">
-      <b-button variant="danger" v-if="requiresAuth" :id="`popover-href-${id}-btn`" @click="handleAuthButton">
+      <b-button variant="danger" v-if="requiresAuth" tabindex="0" :id="`popover-href-${id}-btn`" @click="handleAuthButton">
```

Comment:
Don't we need the tag="a" as per https://bootstrap-vue.org/docs/components/popover/#caveats-with-focus-trigger-on-button-elements?